shared/data/constants.js

```diff
@@ -8,7 +8,8 @@ let constants = {
     "entityMap": "data/tracker_lists/entityMap.json",
     "easylists": ["privacy", "general"],
     "blocking": ["Advertising", "Analytics"],
-    "httpsWhitelist": "data/httpsWhitelist.json",
+    "httpsUpgradeList": "data/httpsUpgradeList.json",
```

Comment:
I'm not sure this is a good idea, because I'm concerned about a user getting this bundled list, having a busted redirect forever loop on a site they care about, and just having a terrible experience. If we think this is really not probable, and our preference is not breaking the web -- we should probably just try again at some point or improve that logic in the next release.

Additionally, I'm not sure that removing the whitelist is ok. @laurengarcia has more knowledge here. I don't think it's safe to say today that we don't need this. I think it's much safer to say that we should have the whitelist come from the server though.

Reply:
Agree, I will remove the local upgrade list.

I don't understand what value the whitelist adds in it's current form. Can't we just remove the entries from the larger list on the server and have it sync without shipping an update?

I could see a case for a whitelist that does updates during browser session since changes to the full list only sync on new browser sessions? But that seems out of scope for this PR.

Reply:
I think you can consider them both problematic, sure. My point is that the whitelist exists and was explicitly made to resolve issues. If that list is for sure incorporated by @zachthompson 's work on the Backend, then you'd be 100% right. I'm not sure we have captured the same edge case information there yet. We can push it as part of the dataset investigations over there though.

Reply:
I added an asana comment to that effect linking back here. Thanks.